bayes_kit/rhat.py

```diff
@@ -27,10 +104,13 @@ def rhat(chains: list[SeqType]) -> FloatType:
     R-hat statistic
 
     Throws:
-    ValueError: if there are fewer than two chains
+    ValueError: If there is not at least one chain or if any chain has
+    fewer than two elements.
```

Comment:
```suggestion
    ValueError: If there is not at least one chain.
    ValueError: If any chain has fewer than two elements.
```

Typically different raise statements are documented separately like this rather than by saying "or"

Reply:
Fixed.  Thanks for letting me know.